package.json

```diff
@@ -85,7 +85,7 @@
     "lodash.pickby": "^4.6.0",
     "openapi3-ts": "^2.0.2",
     "postman-collection": "^4.1.2",
-    "tslib": "^2.3.1",
+    "tslib": "^2.6.2",
```

Comment:
Number.MAX_SAFE_INTEGER and Number.MIN_SAFE_INTEGER are not available on tslib v2.3.1 so we need to update